src/main/kotlin/br/com/zup/op/events/producer/KafkaProducerConfig.kt

```diff
@@ -12,6 +15,8 @@ import java.util.*
 @Configuration
 class KafkaProducerConfig {
 
+  @Autowired
+  private val consumerFactory: ConsumerFactory<String, Any>? = null
 
   //@Value("${spring.kafka.bootstrapAddress}")
```

Comment:
Remove the comment, the `val`, the `KafkaTemplate`

Reply:
The attribute will be reviewed after William will confirm a code publishing event